deploy/canary/Dockerfile

```diff
@@ -3,8 +3,7 @@ MAINTAINER vmarmol@google.com
 
 RUN apt-get install -y git dmsetup
```

Comment:
Ah, that makes sense, sorry I missed that before. Do you think it's better to build cAdvisor in the docker container, like it was before? Or does it not make a difference? I'm OK with whichever you think is best :)